z3rro/src/lit.rs

```diff
@@ -0,0 +1,42 @@
+use crate::interpreted::FuncDef;
+use std::fmt::Debug;
+use z3::ast::{Ast, Dynamic};
+use z3::{Context, Sort};
+
+#[derive(Debug)]
+pub struct LitDecl<'ctx> {
+    _ctx: &'ctx Context,
+    arg_sort: Sort<'ctx>,
+    func: FuncDef<'ctx>,
+}
+
+/// Identity function that is used to mark constants values. They allow for axioms instantiation
```

Comment:
Add this comment on the struct, not on the impl. Say that it is parameterized over the Z3 Sort of the literal.